config/default.yml

```diff
@@ -33,6 +33,8 @@ public:
 
   fetchLimit: '10MB'
 
+  requestTimeoutSeconds: 120
```

Comment:
I set the default to 120 here so there is no behaviour change for self-hosting users, but we could make this a bit lower if we want as this is also the default for dev/test. We can set our own limit for production using `REQUEST_TIMEOUT_SECONDS`.